contrib/deploy.sh

```diff
@@ -5,8 +5,17 @@ export KUBECONFIG="$(kind get kubeconfig-path)"
 echo "Applying namespaces"
 kubectl apply -f ./namespaces.yml
 
-echo $(head -c 16 /dev/urandom| sha256sum | cut -d " " -f 1) > ./password.txt # Store password in password.txt file
-PASSWORD=$(cat ./password.txt)
+sha_cmd="sha256sum"
```

Comment:
@zeerorg FYI this broke on MacOS so I've put a fix in that @rgee0 added in the faas-cli.

Reply:
I should've taken more care with sha256 utility, thanks for fixing this.